Add tableau (#2469)
timescale/docs: use-timescale/integrations/observability-alerting/tableau.md

---
title: Using Tableau to visualize data in TimescaleDB
excerpt: Use Tableau to plot and visualize your data
products: [cloud, mst, self_hosted]
keywords: [visualizations, analytics, Tableau]
---

# Using Tableau to visualize data in TimescaleDB

[Tableau][get-tableau] is a popular analytics platform that helps you gain
greater intelligence about your business. It is an ideal tool for visualizing
data stored in Timescale.

This section covers:

*   Setting up Tableau to work with TimescaleDB
*   Running queries on TimescaleDB from within Tableau

> *   Visualize data in Tableau

Reviewer: I am not sure if we have this details in the docs yet.

## Prerequisites

Before you begin, make sure you have:

*   Created a [Timescale][cloud-login] service.
*   Signed up for [Tableau][get-tableau].

> *   Found the connection details for the database you want to use as a data
>     source. The details are contained in the cheatsheet you downloaded when you
>     created the service.

Reviewer: ```suggestion
*   Downloaded the cheatsheet when you created the service. This sheet contains the 
     connection details for the database you want to use as a data source.
```
